WheelOfFortune/Prompt.cs

```diff
@@ -29,7 +29,21 @@ public static string CreatePlayer()
         /// </summary>
         public static char AddAdditionalPlayer()
         {
-            return ' ';
+            Console.Write("\nWould you like to add an additional player? (y/n): ");
+            string answer = Console.ReadLine();
+
+            bool isValidInput = answer.ToLower() == "y" || answer.ToLower() == "n";
+
+            while(!isValidInput)
+            {
+                InvalidInputError error = new InvalidInputError((InvalidInputError.ErrorTypes)2);
+                Console.WriteLine($"\n{error.ErrorMessage}");
+                Console.Write("Would you like to add an additional player? (y/n): ");
+                answer = Console.ReadLine();
+                isValidInput = answer.ToLower() == "y" || answer.ToLower() == "n";
+            }
```

Comment:
I think this might be a stylistic preference, but if you set the while conditional as `answer.ToLower() != "y" && answer.ToLower() != "n"` , you could remove lines 35 and 43. Having that variable might make the code a bit more readable though, so your call!

Reply:
great idea! will implement and update pr